docs/docs/target/container_image.md

```diff
@@ -500,3 +500,10 @@ You can configure Docker daemon socket with `DOCKER_HOST` or `--docker-host`.
 ```shell
 $ trivy image --docker-host tcp://127.0.0.1:2375 YOUR_IMAGE
 ```
+
+### Configure Podman daemon socket to connect to.
+You can configure Podman daemon socket with `PODMAN_HOST` or `--podman-host`.
```

Comment:
It looks like `PODMAN_HOST` doesn't work

Reply:
Hey @DmitriyLewen thanks for reviewing the PR, I have added the missing test case as well as support for taking the PODMAN_HOST. Please do let me know if there are any fixes required.